package.json

```diff
@@ -9,11 +9,11 @@
   "description": "Prepare Brotli-compressed versions of assets to serve them with Content-Encoding: br",
   "dependencies": {
     "async": "~2.6.0",
-    "webpack-sources": "^1.0.2"
+    "webpack-sources": "^1.3.0"
   },
   "optionalDependencies": {
-    "brotli": "^1.3.1",
-    "iltorb": "^2.0.1"
+    "brotli": "^1.3.2",
+    "iltorb": "^2.4.0"
```

Comment:
The module here is referenced with ^ this means that latest version until 3.x.x will be used. So why do we need this update?